airbyte-integrations/connectors/destination-snowflake/src/test-integration/kotlin/io/airbyte/integrations/destination/snowflake/typing_deduping/AbstractSnowflakeTypingDedupingTest.kt

```diff
@@ -27,13 +27,12 @@ import kotlin.concurrent.Volatile
 import org.junit.jupiter.api.Assertions
 import org.junit.jupiter.api.Assertions.assertEquals
 import org.junit.jupiter.api.Assertions.assertThrows
-import org.junit.jupiter.api.Disabled
 import org.junit.jupiter.api.Test
 
 private val LOGGER = KotlinLogging.logger {}
 
 abstract class AbstractSnowflakeTypingDedupingTest : BaseTypingDedupingTest() {
-    private var databaseName: String? = null
+    private lateinit var databaseName: String
```

Comment:
switching to lateinit b/c otherwise the compiler complains about `databaseName` potentially changing values between its assignment on line 53 and its usage in `cleanAirbyteInternalTable` on 57 >.>

Reply:
This spotbugs thing is really confusing when it is ok with lateinit and when it is not (I need to figure out how to logically reason about this)

Reply:
I still have no idea why spotbugs yells at everyone else about lateinit, I've never had it happen to me >.>